src/main/java/in/projecteka/consentmanager/consent/ConsentManager.java

```diff
@@ -57,9 +59,18 @@ private static boolean isNotSameRequester(ConsentArtefact consentDetail, String
                 .flatMap(context -> validatePatient(requestedDetail.getPatient().getId(), context.get("Authorization")))
                 .then(validateHIPAndHIU(requestedDetail))
                 .then(saveRequest(requestedDetail, requestId))
+                .then(sendConsentRequestNotification(requestId, requestedDetail))
                 .thenReturn(requestId);
     }
 
+    private Mono<Void> sendConsentRequestNotification(String requestId, RequestedDetail requestedDetail) {
```

Comment:
```suggestion
    private Mono<Void> sendConsentRequestNotification(String requestId, RequestedDetail requestedDetail) {
```

No need of this method I believe.